src/edictor/app/js/glosses.js

```diff
@@ -519,37 +520,37 @@ GLOSSES.markID = function(event, node) {
     // now go as normal
     if (node.dataset['marked'] == "1"){
       node.dataset['marked'] = "0";
-      node.style.backgroundColor = node.parentNode.style.backgroundColor;
+      node.classList.remove('id_batch');
       GLOSSES.joined[node.dataset['idx']] = false;
     }
     else {
       node.dataset['marked'] = "1";
-      node.style.backgroundColor = "Salmon";
+      node.classList.add('id_batch');
       GLOSSES.joined[node.dataset['idx']] = true;
     }
   }
 };
 
 GLOSSES.markIDs = function(event, node) {
   event.preventDefault();
-  if (node.dataset["idx"] in this.joined) {
-    for (nodeidx in this.joined) {
-      this.markID(event, document.getElementById('GLOSSES_idx-' + nodeidx));
-    }
-    this.joined = {};
-    return;
-  }
+  // if (node.dataset["idx"] in this.joined) {
```

Comment:
So the behaviour would rather be: if you click left mouse in a field that is marked, unselect all, if you click in a field not marked, just ignore. Would that make sense to you, @patkaiist ?

Reply:
From what you showed previously, the lightblue colouring does not seem to have a functional difference from the salmon/default colouring distinction. The reason for repurposing right click is that if you over-select, it's tedious to remove them one at a time. Since control click covers the previous right-click functionality but in a more controlled way, it seemed useful to offer a way to deselect _en masse_ as well.

> So the behaviour would rather be: if you click left mouse in a field that is marked, unselect all, if you click in a field not marked, just ignore

I'm not sure I follow this.

Reply:
I merge and we can adjust later, if needed, I'd remove the `//` if I am happy after testing :-)